adafruit_motor/stepper.py

```diff
@@ -113,8 +113,13 @@ def __init__(
             # set a safe pwm freq for each output
             self._coil = (ain2, bin1, ain1, bin2)
             for i in range(4):
-                if self._coil[i].frequency < 1500:
+                if self._coil[i].frequency < 1500 and self._coil[i].variable_frequency:
                     self._coil[i].frequency = 2000
+                else:
+                    raise RuntimeError(
```

Comment:
This is more of a "bad arguments" error than some uncontrollable failure at run-time, so I suggest using `ValueError`.

```suggestion
                    raise ValueError(
```

Reply:
Good idea, that's a better fit.